### PR TITLE
[Feature:Developer] Add install lichen script alias

### PR DESCRIPTION
**Pull request by sbelsk**
### Please check if the PR fulfills these requirements:
* [ ] Documentation has been updated/added if relevant

### What is the current behavior?
Currently, when developing/working on the Lichen repo, to install lichen you have to run `bash /usr/local/submitty/GIT_CHECKOUT/Lichen/install_lichen.sh` (or `install_submitty` which calls the latter but installs many other things too) inside the vagrant terminal.

### What is the new behavior?
An additional saved variable and alias were added to make Lichen development more convenient :)

### CODE REVIEW

File: .setup/install_system.sh [hunk 1]
```diff
@@ -135,6 +139,8 @@ export DAEMONCGI_GROUP=${DAEMONCGI_GROUP}
 alias submitty_help=\"echo -e '${INSTALL_HELP}'\"
 alias install_submitty='/usr/local/submitty/.setup/INSTALL_SUBMITTY.sh'
 alias submitty_install='/usr/local/submitty/.setup/INSTALL_SUBMITTY.sh'
+alias install_lichen='sudo bash /usr/local/submitty/GIT_CHECKOUT/Lichen/install_lichen.sh'
+alias lichen_install='sudo bash /usr/local/submitty/GIT_CHECKOUT/Lichen/install_lichen.sh'
```
Comment (MasterOdin) [inline .setup/install_system.sh:143]: You don't need `sudo` here. These aliases are only available to the root user.

**Review by MasterOdin — CHANGES_REQUESTED** [review]

File: .setup/install_system.sh (hunk 1 above)
Comment (sbelsk) [reply .setup/install_system.sh:143]: I tried it without the `sudo` first and it spit out an error about missing permissions, so I added it...

Comment (MasterOdin) [reply .setup/install_system.sh:143]: What was the error message?

Comment (sbelsk) [reply .setup/install_system.sh:143]: Hm, I don't remember but I can try testing it again. Btw, is the only way to test this script to do `destroy` and `up` in vagrant, or is it possible to somehow rerun it?

Comment (MasterOdin) [reply .setup/install_system.sh:143]: You can just directly edit `/root/.bashrc` and `source /root/.bashrc` to test the aliases.

**Review by MasterOdin — APPROVED** [review]

**Pull request merged**